[Chore#301] Update technologyType to remove array-specific details

ke4 opened the pull request:
Removed array-specific references from the "technologyType" field. This enhances consistency and simplifies the dataset for further analysis.

ke4 committed:
Update technologyType to remove array-specific details

Removed array-specific references from the "technologyType" field. This enhances consistency and simplifies the dataset for further analysis.

Signed-off-by: Karoly Erdos <karoly@ebi.ac.uk>

amnonkhen commented on package.json:
```diff
@@ -2,8 +2,8 @@
-    "webpack-cli": "^5.1.4",
-    "@ebi-gene-expression-group/highcharts-series-generator": "file:packages/highcharts-series-generator"
+    "webpack-cli": "^5.1.4"
   }
```
I would make sure this file has an "engines" attribute that limits to the app's node version (14)

ke4 committed:
Refactor data processing chain into a reusable function

Simplified the logic for processing filter options by refactoring repeated code into a reusable function. The new approach improves readability and maintains functionality, with dynamic capitalization applied only when appropriate.

Signed-off-by: Karoly Erdos <karoly@ebi.ac.uk>

ke4 commented on packages/atlas-experiment-table/src/TableManager.js:
```diff
@@ -93,14 +110,7 @@ export default class TableManager extends React.Component {
-              .flatMap(dropdownFilter.dataKey)
-              .map(_.toString)
-              .map(_.trim)
-              .map(capitalizeEachWord)
```
I agree, that we should simplify this method, but your code is doing the opposite, I believe ;-)
Anyway, I solved it another way.

ke4 commented:
I cleaned the code.
The other comments can be fixed in another ticket where we do it for all our components if it is really necessary.
